test/src/main_receiver.cpp

```diff
@@ -26,36 +37,69 @@ using hclock = chrono::high_resolution_clock;
 using TIME = NDTime;
 
 
-/***** SETING INPUT PORTS FOR COUPLEDs *****/
+/**
+ * Setting input ports for messages
+ */
 struct inp : public cadmium::in_port<message_t> {};
 
-/***** SETING OUTPUT PORTS FOR COUPLEDs *****/
+/**
+ * Setting output ports for messages 
+ */
 struct outp : public cadmium::out_port<message_t> {};
 
 
-/********************************************/
-/****** APPLICATION GENERATOR *******************/
-/********************************************/
+/**
+ * This is application generator class takes file path
+ * parameter and waits for input
+ * @tParam message T
+ */
 template<typename T>
 class ApplicationGen : public iestream_input<message_t,T> {
     public:
         ApplicationGen() = default;
-        ApplicationGen(const char* file_path) : iestream_input<message_t,T>(file_path) {}
+
+        /**
+         * A parameterized contructor for class application generator 
```

Comment:
it will be constructor

Reply:
and contains

Reply:
there are more spelling mistakes in line 155 and 99 and 200

Reply:
there are more spelling mistakes in sender and subnet .cpp files

Reply:
fixed spelling mistakes in all the mentioned files.